Rakefile

```diff
@@ -4,6 +4,7 @@ default_tasks = []
 
 RSpec::Core::RakeTask.new(:spec) do |spec|
   spec.pattern = 'spec/**/*_spec.rb'
+  spec.rspec_opts = "--tag ~e2e" if ENV["RUN_E2E_TESTS"] != "true"
```

Comment:
We should move this to Travis (like the runscope token). I believe non-owners won't be able to get the Runscope token anyway.

Reply:
I've added RUN_E2E_TESTS as a variable in travis https://cloudup.com/cmDLAS18POA

Reply:
I placed this check here to ensure that `bundle exec rake` works on a local system without hitting the network - less friction for an external contributor trying to run tests on their machine. 

> I believe non-owners won't be able to get the Runscope token anyway.

Ah, you're right - I've changed this to only run the end-to-end tests if travis secure env vars are present (master builds, or PRs from owners). https://github.com/segmentio/analytics-ruby/pull/137/commits/6b2983ce92c4ed72a8a037cfb12f26e326d67c78